modules/nwt-gateway-mode.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="nwt-gateway-mode_{context}"]
+= Setting local and shared gateway modes
+
+As a cluster administrator you can configure the gateway mode using the `gatewayConfig` spec in the Cluster Network Operator. The following procedure can be used to set the `routingViaHost` field to `true` for local mode or  `false` for shared mode.
+
+You might follow the optional step 4 to enable IP forwarding alongside local gateway mode if you need the host network of the node to act as a router for traffic not related to OVN-Kubernetes. For example, possible use cases for combining local gateway mode with IP forwarding include:
```

Comment:
Per the ISG, "[might](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#m)" is used to indicate possibility ("this might happen"), while "[can](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#c)" is used to indicate ability ("You can do this action, if you want"). Do you think this is a case where you could use "can" to indicate that someone has the option to do this? 

```suggestion
You can follow the optional step 4 to enable IP forwarding alongside local gateway mode if you need the host network of the node to act as a router for traffic not related to OVN-Kubernetes. For example, possible use cases for combining local gateway mode with IP forwarding include:
```

I don't know the context here so I will leave it up to you

Reply:
For the best of my knowledge, "can" would make more sense as per your definition. But I'll let @JoeAldinger decide.